app/models/miq_schedule_worker/jobs.rb

```diff
@@ -120,7 +120,6 @@ def archived_entities_purge_timer
     queue_work(:class_name => "ContainerGroup", :method_name => "purge_timer", :zone => nil)
     queue_work(:class_name => "ContainerImage", :method_name => "purge_timer", :zone => nil)
     queue_work(:class_name => "ContainerProject", :method_name => "purge_timer", :zone => nil)
-    queue_work(:class_name => "ContainerDefinition", :method_name => "purge_timer", :zone => nil)
```

Comment:
@cben @zeari I suppose these jobs on the queue can be deleted in the migration? (just to avoid confusing errors... I saw that BZ are filed when there are scary errors)